lblod/frontend-reglementaire-bijlage
# GN-4345 Cleanup code: consistent routing, remove buttons and translations

## Overview
This fixes three code-related issues:
- cleanup the routes so regulatory attachments are also under a subroute and all routes use the order `/:id/crud-action`
- Together with these route changes, the breadcrumbs are fixed
- all remove buttons in lists are consistent `[trash-icon] Remove` style
- translations are consistent for "regulatory attachments" and "fragmentenlijst(en)" (before sometimes "regulatory statements" and "fragmentlijst(en)"). **Note**: there is no certainty if `attachments` or `statements` is the correct term. This PR chooses one (arbitrarily) just to make it at least consistent across the app.


##### connected issues and PRs:
https://binnenland.atlassian.net/browse/GN-4345

### How to test/reproduce
- navigate across all pages and check if clicking breadcrumbs work (and they are correct)
- see all "remove" buttons being the same style
- check if nothing broke because of moving files.

### Challenges/uncertainties
- attachments <-> statements is not a clear distinction. But it should at least be consistent, which was fixed here.
- this PR does a lot of small things together. The most important part is the route changes (and related breadcrumbs). Reviewing per commit might be an idea if everything together is too big of a change.

- some tests failed because of file moves. They didn't really test anything so I just removed them. I did leave the sparql endpoint test, because testing that the endpoint stays in the same route actually is really sensible. (small note: accidentally pushed this changed on master first. I reverted and pushed it to this branch instead :) )
